src/components/Icon/index.js

```diff
@@ -34,13 +41,34 @@ class Icon extends PureComponent {
         const width = this.props.small ? variables.iconSizeSmall : this.props.width;
         const height = this.props.small ? variables.iconSizeSmall : this.props.height;
         const IconToRender = this.props.src;
-        return (
+
+        const rendered = (
             <IconToRender
                 width={width}
                 height={height}
                 fill={this.props.fill}
             />
         );
+
+        if (this.props.inline) {
+            return (
+                <View style={[{
+                    width, height,
+                }, styles.iconInlineWrapper]}
+                >
+                    <View style={[{
+                        width,
+                        height,
+                        top: inlineTop,
```

Comment:
No inline styles...
https://github.com/Expensify/App/blob/2aaa8fd10bec0cfb6346140595a5f55540edacd9/src/styles/StyleUtils.js#L230

Reply:
@parasharrajat what's the advantage of using this function as it offers no memoization? Or it's just a style code thing?

Reply:
Code Style thing...